jupyterlab_gitplus/__init__.py

```diff
@@ -1,6 +1,9 @@
 
 import json
 from pathlib import Path
+from urllib.parse import urlparse
+import logging
+logger = logging.getLogger(__name__)
 
 from ._version import __version__
```

Comment:
My local build kept failing until I removed this statement. 

Not related to your change but it seems like the original state was broken i.e. it's referring to `._version` which does not exist & is not even required since we explicitly set the version on line 31 below this.

Reply:
In my *installation* of jupyterlab gitplus, `_version.py` exists and has these contents:

```
import json
from pathlib import Path

__all__ = ["__version__"]

def _fetchVersion():
    HERE = Path(__file__).parent.resolve()

    for settings in HERE.rglob("package.json"): 
        try:
            with settings.open() as f:
                return json.load(f)["version"]
        except FileNotFoundError:
            pass

    raise FileNotFoundError(f"Could not find package.json under dir {HERE!s}")

__version__ = _fetchVersion()
```

The actual version info is in `jupyterlab_gitplus/labextension/package.json`. So I'm guessing you are trying to run the code directly from the git repo, while this stuff gets added in by the jupyterlab extension packaging system. 

I'll leave it alone for now, but can remove it if you want to override ⬆️  behavior (or if my guess as to what's happening is wrong).